tests/checkup_test.go

```diff
@@ -178,6 +208,40 @@ func newKiagnoseConfigMapAccessRoleBinding() *rbacv1.RoleBinding {
 	}
 }
 
+func newKubeVirtRTCheckerRole() *rbacv1.Role {
+	return &rbacv1.Role{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: testKubeVirtRTCheckerRoleName,
+		},
+		Rules: []rbacv1.PolicyRule{
+			{
+				APIGroups: []string{"kubevirt.io"},
+				Resources: []string{"virtualmachineinstances"},
+				Verbs:     []string{"create"},
+			},
+		},
+	}
+}
+
+func newKubeVirtRTCheckerRoleBinding() *rbacv1.RoleBinding {
```

Comment:
It makes sense (to me), to express the relation between the role and the role-binding, such that the role-binding will accept as input the role itself (or its name) with the SA.

In order to understand this relation, we need to travel through the constants, which makes this thing fragile IMO.

Reply:
Done.

Reply:
I'll create another PR to apply this change for the other objects.